assets/common.css

```diff
@@ -52,7 +55,6 @@
 
 * {
   box-sizing: border-box;
-  transition: outline 0s;
```

Comment:
This is used to prevent the weird animations when navigating using the keyboard

Reply:
Transition does not play in this case so there is no point keeping it here. The border is still visible when navigating on keyboard.
![image](https://user-images.githubusercontent.com/49114212/201516188-03daeff6-1522-4c7a-b7e1-1663235244e8.png)